MiniDebug.cs

```diff
@@ -327,4 +332,33 @@ private void DestroyLoadScreens()
             Destroy(obj);
         }
     }
+
+    private void OnPostRenderCallback(Camera cam)
+    {
+        if (cam == GameCameras.instance.cameraController.cam)
+        {
+            if (cameraControllerPosition != null)
+            {
+                GameCameras.instance.cameraController.transform.position = cameraControllerPosition.Value;
+                cameraControllerPosition = null;
+            }
+        }
+    }
+
+    private void OnPreRenderCallback(Camera cam)
+    {
+        if (cam == GameCameras.instance.cameraController.cam)
+        {
+            if (GameManager.instance.IsNonGameplayScene())
+            {
+                return;
+            }
+            if (CameraFollow && GameManager.instance.hero_ctrl is { } heroCtrl)
```

Comment:
Any specific reason for accessing hero controller through GM instead of doing `HeroController.instance`?

Reply:
There is no reason, the HeroController.instance is probably better